eusurdf/euslisp/eusmodel_to_urdf.l

```diff
@@ -42,6 +42,8 @@
                          mesh-out-dir
                          model-name
                          urdf-out-path))
+    (unix:system (format nil "rm -f ~A"
+                         (send (concatenate-pathnames tmpdir model-name ".dae") :namestring)))
     (unix:system (format nil "sed -i -e \"s@continuous@revolute@g\" ~A" urdf-out-path))
```

Comment:
Sometimes user does not have permission to remove files in /tmp directory , which other users generated. 
Please see https://github.com/jsk-ros-pkg/jsk_model_tools/issues/124 again.
I think tmp-dir-no-duplication option for `(irteus2urdf-room-for-gazebo)` is not necessary and it is ok to always generate no duplication directory: https://github.com/jsk-ros-pkg/jsk_model_tools/pull/188/files#diff-fd3566e2a31868b72476ad66c9e5e01dR55 .
So please remove following section and move it to `(eusmodel2urdf)` : https://github.com/jsk-ros-pkg/jsk_model_tools/pull/188/commits/d6812feb80bc5bfeb8a8c425f820725bc54f4481#diff-fd3566e2a31868b72476ad66c9e5e01dR72